src/org/zaproxy/zap/extension/scripts/dialogs/NewScriptDialog.java

```diff
@@ -168,13 +169,20 @@ private void resetTemplates() {
 			list.add("");
 		}
 		list.addAll(extension.getExtScript().getScriptingEngines());
+		// Remove the null scripting engine - unfortunately there is no easy way to do this
+		// and ExtensionScript.LANG_ENGINE_SEP (" : ") is private
+		list.remove(" : " + NullScriptEngine.NAME);
```

Comment:
`NullScriptEngineWrapper.NAME`? This (and `getTypes` change) also needs to be done in `LoadScriptDialog`.